.changelog/_unreleased.toml

```
@@ -0,0 +1,5 @@
+[[entries]]
+id = "c735fee0-cd87-47a6-8c6f-b52463f0f73e"
+type = "feature"
+description = "Add flags to CargoClippyTask to cover more cases"
+author = "@tomkarw"
```

```suggestion
author = "@tomkarw"
pr = "https://github.com/kraken-build/kraken/pull/182"
```

Unfortunately the automation to inject this field is broken atm. 😄